deploy/olm-catalog/integreatly-operator/integreatly-operator-1.18.0/integreatly-operator.v1.18.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,368 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RHMI",
+          "metadata": {
+            "name": "INSTALLATION_NAME"
+          },
+          "spec": {
+            "namespacePrefix": "INSTALLATION_PREFIX-",
+            "operatorsInProductNamespace": "OPERATORS_IN_PRODUCT_NAMESPACE",
+            "selfSignedCerts": "SELF_SIGNED_CERTS",
+            "smtpSecret": "INSTALLATION_PREFIX-smtp",
+            "type": "INSTALLATION_TYPE",
+            "useClusterStorage": "USE_CLUSTER_STORAGE"
+          }
+        },
+        {
+          "apiVersion": "integreatly.org/v1alpha1",
+          "kind": "RHMI",
+          "metadata": {
+            "name": "example-rhmi"
+          },
+          "spec": {
+            "namespacePrefix": "redhat-rhmi-",
+            "selfSignedCerts": true,
+            "smtpSecret": "redhat-rhmi-smtp",
+            "type": "managed",
+            "useClusterStorage": true
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Integreating greatly
+    certified: "false"
+    containerImage: quay.io/integreatly/integreatly-operator:v1.18.0
+    createdAt: "2019-10-15 10:39:00"
+    description: Greatly integrating integreatly greatness
+    support: Integreatly
+  name: integreatly-operator.v1.18.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: RHMI
+      name: rhmis.integreatly.org
+      version: v1alpha1
```

Comment:
Becasue there kind is now `RHMI`, the csv generation looks to be missing the `displayName`, `description` and `specDescriptors` fields from the previous csv. I think OLM installation would fail due these missing fields 
https://github.com/integr8ly/integreatly-operator/blob/a8cfbe351d92627d7c7dd5dda270225656c3158d/deploy/olm-catalog/integreatly-operator/integreatly-operator-1.17.0/integreatly-operator.v1.17.0.clusterserviceversion.yaml#L49
https://github.com/integr8ly/integreatly-operator/blob/a8cfbe351d92627d7c7dd5dda270225656c3158d/deploy/olm-catalog/integreatly-operator/integreatly-operator-1.17.0/integreatly-operator.v1.17.0.clusterserviceversion.yaml#L50
https://github.com/integr8ly/integreatly-operator/blob/a8cfbe351d92627d7c7dd5dda270225656c3158d/deploy/olm-catalog/integreatly-operator/integreatly-operator-1.17.0/integreatly-operator.v1.17.0.clusterserviceversion.yaml#L53

Reply:
Thanks Kevin I've added these now.